apps/condo/domains/ticket/utils/clientSchema/search.ts

```diff
@@ -20,7 +20,7 @@ const GET_ALL_SOURCES_QUERY = gql`
 
 // TODO(pahaz): add organization relation to existing classifiers
```

Comment:
It seems, like `organisation` relation was removed from `TicketClassified` and this comment is now orphaned.

Reply:
The future of classifiers is undefined, so I shall not remove this todo